pypyr-go-terra/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# note ENTRYPOINT is still pypyr, meaning you can run this container as if it's
+# the pypyr binary.
+# be aware that the WORKDIR is NOT the GOPATH.
+# the WORKDIR is /src, which is the pypyr working dir. this will let you mount a
+# volume with a pipelines dir in it without hiding the go path.
+# build me like this from repo root:
+# docker build -t pypyr/pypyr-go-terraform -t pypyr/pypyr-go-terraform:latest -t pypyr/pypyr-go-terraform:0.11 -t pypyr/pypyr-go-terraform:0.11.8 -f pypyr-go-terraform/Dockerfile .
+
+FROM pypyr/pypyr-go:pypyr1.0.0-go1.11
+
+ARG terraformversion=0.11.8
+
+# get the Terraform executable as zip
```

Comment:
not a useful comment - it just describes what the 1st line wget does. think the code is clear enough you can just get rid of comment,